src/UnoFlipController.java

```diff
@@ -37,8 +33,10 @@ public UnoFlipController(UnoFlipModel model){
 
     @Override
     public void actionPerformed(ActionEvent e) {
-        if (e.getActionCommand().equals(UnoFlipViewFrame.START_CMD)){
-
+        // Start game command to initialize players
```

Comment:
I would suggest writing a short high level explanation of what is to happen in this conditional block. Somethign like "the user started the game, set up the game using parameters obtained from user input", so that for example if someone was just skimming through to find a specific section of code, they could easily follow without needing to read the details (almost like function headers/javadoc for large-ish blocks of code)